br/pkg/restore/systable_restore.go

```diff
@@ -71,6 +75,72 @@ func isStatsTable(tableName string) bool {
 	return ok
 }
 
+// ClearSystemUsers is used for volume-snapshot restoration.
+// because we can not support restore user in some scenarios, for example in cloud.
+// we'd better use this function to drop cloud_admin user after volume-snapshot restore.
+func (rc *Client) ClearSystemUsers(ctx context.Context, filterUsers []string) error {
+	sysDB := mysql.SystemDB
+	db, ok := rc.getDatabaseByName(sysDB)
+	if !ok {
+		log.Warn("target database not exist, aborting", zap.String("database", sysDB))
+		return nil
+	}
+
+	execSQL := func(sql string) error {
+		// SQLs here only contain table name and database name, seems it is no need to redact them.
+		if err := rc.db.se.Execute(ctx, sql); err != nil {
+			log.Warn("failed to execute SQL restore system database",
+				zap.Stringer("database", db.Name),
+				zap.String("sql", sql),
+				zap.Error(err),
+			)
+			return berrors.ErrUnknown.Wrap(err).GenWithStack("failed to execute %s", sql)
+		}
+		log.Info("successfully restore system database",
```

Comment:
I think this log can be more clear, for example "successfully execute SQL to clear system user". So is the error log.

Reply:
fixed in https://github.com/pingcap/tidb/pull/39986/commits/a1f401fc4c2894aa7086ca3e2270c30d65453fa2